src/renderer/containers/Table/CustomCells/StatusCell/index.tsx

```diff
@@ -99,7 +99,9 @@ export default function StatusCell(props: CellProps<UploadSummaryTableRow>) {
       step = Step.TWO;
       displayForStep = "0";
       totalForStep = "1";
-    } else if (bytesUploaded && totalBytes) {
+    } 
+    // If uploading bytes, and progress has been made, display progress
```

Comment:
Unrelated to focus of pr.  An improvement made on wrong branch during an separate pr review.

Reply:
The formatting of this is weird, could you move it down below the if?

Reply:
pushed a change, lmk if it is good now.